trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/induced_drag_aircraft.py

```diff
@@ -53,7 +53,7 @@ def induced_drag_aircraft(state,settings,geometry):
     e             = configuration.oswald_efficiency_factor
     K             = configuration.viscous_lift_dependent_drag_factor
     wing_e        = geometry.wings['main_wing'].span_efficiency
-    ar            = geometry.wings['main_wing'].aspect_ratio # TODO: get estimate from weissinger

```

Comment:
I think the note was meant for span_efficiency. Maybe make a github issue

Reply:
Will do. Issue Resolved